tests/Test.hs

```diff
@@ -295,32 +295,6 @@ moduleTests = testGroup "Module tests" [ shouldWork, shouldn'tWork ]
                     other -> assertFailure ("Unexpected thing def " ++ show other)
                 _ -> assertFailure "Expected an import, two macros, a definition, and examples"
           )
-        , ( "examples/hygiene.kl"
-          , \m _ ->
-              view moduleBody m & map (view completeDecl) &
-              \case
-                [Import _, Import _, Define _ fun1 _ firstFun, DefineMacros [_], Define _ fun2 _ secondFun,
-                 Example _ _ e1, Example _ _ e2, DefineMacros [_], Example _ _ e3] -> do
-                  spec1 <- lam \x -> lam \_y -> x
-                  spec2 <- lam \_x -> lam \y -> y
-                  assertAlphaEq "First fun drops second argument" firstFun spec1
-                  assertAlphaEq "Second fun drops first argument" secondFun spec2
-                  case e1 of
-                    Core (CoreApp (Core (CoreApp f _)) _) ->
-                      assertAlphaEq "Ex 1 picks fun 2" (Core (CoreVar fun2)) f
-                    other -> assertFailure $ "Ex 1 should be an application, but it's " ++ shortShow other
-                  case e2 of
-                    Core (CoreApp (Core (CoreApp f _)) _) ->
-                      assertAlphaEq "Ex 2 picks fun 1" (Core (CoreVar fun1)) f
-                    other -> assertFailure $ "Ex 2 should be an application, but it's " ++ shortShow other
-                  spec3 <- lam (const (pure (Core (CoreVar fun2))))
-                  case e3 of
-                    Core (CoreApp (Core (CoreApp (Core (CoreApp f _)) _)) _) ->
-                      assertAlphaEq "Ex 3 picks fun 2" f spec3
-                    other -> assertFailure $ "Ex 3 should be an application, but it's " ++ shortShow other
-
-                _ -> assertFailure "Expected two imports, a def, a macro, a def, two examples, a macro, and an example"
-          )
```

Comment:
@david-christiansen this test is no longer passing, because I added an import and it expects exactly 2 imports. I would like to delete the test instead of fixing it, because it's harder to maintain than the golden tests, and I think it's just a leftover from when we didn't yet have the golden test system in place. Do you agree? If so, should I take the opportunity to delete more of those non-golden tests?

One reason I hesitate is that this test tests something different from the corresponding golden test: it doesn't check that the program's output is `'b`, `'a`, and `'b`, as we'd expect if hygiene is respected, instead it checks that the fully-expanded code is as expected, which is a lot more precise.

Reply:
Never mind, I've changed my mind about this PR, I no longer want to delete this test.